src/fpm_command_line.f90

```diff
@@ -1459,6 +1536,30 @@ subroutine set_help()
     ' --all             Delete the build without prompting including dependencies.', &
     ' --registry-cache  Delete registry cache.', &
     '' ]
+    help_search=[character(len=80) :: &
+    'NAME', &
+    ' search(1) - search for the package in local and fpm - registry.', &
+    '', &
+    'SYNOPSIS', &
+    ' fpm search', &
+    '', &
+    'DESCRIPTION', &
+    ' Search for packages in the local directory and the fpm-registry, ', &
+    ' supports package search by name, namespace, query (description and README.md)', &
+    ' and license from the registries (local and remote).', &
+    '', &
+    'OPTIONS', &
+    ' --query              Search query (supports wildcard).', &
```

Comment:
what does the `--query` option do? It's not clear to me from either the help or the code. Should we try to find a better name for it?

Reply:
it supports any term/keyword search in the description of the fpm.toml and for fpm-registry it supports search across all parameters like name, namespace,version, keywords, README. I am also not very sure if its a suitable name.

Reply:
Thanks @henilp105, I see. Well I think it would be necessary to have a list of all possible query terms, like :
- dependencies
- version
- build
- maintainer
- etc.